### PR TITLE
koord-manager: modify quota-webhook

### PR DESCRIPTION
**Pull request by xulinfei1996**
Signed-off-by: xulinfei.xlf <xulinfei.xlf@alibaba-inc.com>

### Ⅰ. Describe what this PR does
The main purpose is to reorganize the InjectCache function of the webhooks, only allow validate-quota to addEventHandlers.

### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,10 +4,10 @@ GIT_BRANCH ?= $(shell git rev-parse --abbrev-ref HEAD)
 GIT_COMMIT_ID ?= $(shell git rev-parse --short HEAD)
 
 # Image URL to use all building/pushing image targets
-REG ?= ghcr.io
```
Comment (jasonliu747) [inline Makefile:7]: please remove this from your commit.

**Review by jasonliu747 — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (xulinfei1996) [reply Makefile]: fixed

**Review by buptcozy — APPROVED** [review]
/LGTM

**Review by eahydra — APPROVED** [review]
/lgtm
/approve

**Review by buptcozy — APPROVED** [review]
/lgtm

**Review by jasonliu747 — APPROVED** [review]
/lgtm

**Pull request merged**